pkgs/servers/sql/postgresql/default.nix

```diff
@@ -22,14 +22,18 @@ let
 
     makeFlags = [ "world" ];
 
+    preConfigure = ''
+    export CFLAGS="$CFLAGS -I${libxml2.dev}/include/libxml2"
```

Comment:
I think we usually prefer to do this like:

```
NIX_CFLAGS_COMPILE = "-I${libxml2.dev}/include/libxml2";
```

Mostly this is because not all build systems recognize CFLAGS.